plugins/importexport/native/native.xsd

```diff
@@ -80,12 +97,13 @@
 			<complexContent>
 				<extension base="pkp:pkppublication">
 					<sequence>
+						<element ref="pkp:series" maxOccurs="1" minOccurs="0" />
 						<element ref="pkp:chapters" maxOccurs="1" minOccurs="0" />
 						<element ref="pkp:covers" minOccurs="0" maxOccurs="1" />
 						<element ref="pkp:citations" minOccurs="0" maxOccurs="unbounded"/>
 					</sequence>
 					<attribute name="series" type="normalizedString" />
-					<attribute name="series_position" type="normalizedString" />
+					<attribute name="seriesPosition" type="normalizedString" />
```

Comment:
I'd recommend leaving this as it was -- it's an easily avoided breaking change.

Reply:
Fixed.